geomstats/geometry/grassmannian.py

```diff
@@ -50,30 +50,174 @@ def __init__(self, n, k):
         self.k = k
         self.metric = GrassmannianCanonicalMetric(3, 2)
 
-    def belongs(self, point, tolerance=TOLERANCE):
+    def random_uniform(self, n_samples=1):
+        """Sample random points from a uniform distribution.
+
+        Following [Chikuse03]_, :math: `n_samples * n * k`scalars are sampled
+        from a standard normal distribution and reshaped to matrices,
+        the projectors on their first k columns follow a uniform distribution.
+
+        Parameters
+        ----------
+        n_samples: int,
```

Comment:
NIT: space before : , no comma after int, "." at the end of parameters' description